smart_proxy_test.go

```diff
@@ -65,7 +65,7 @@ func TestStartServer_BadServerAddress(t *testing.T) {
 // TestPrintVersionInfo is dummy ATM - we'll check versions etc. in integration tests.
 // TODO: add check for actual messages that are printed to standard output
 func TestPrintVersionInfo(t *testing.T) {
-	main.PrintVersionInfo()
+	assert.Equal(t, main.ExitStatusOK, int(main.PrintHelp()))
```

Comment:
wrong command? it should be `PrintVersionInfo` not `PrintHelp`

Reply:
it should I guess :)

Reply:
fixed